src/main/groovy/com/jakewharton/sdkmanager/internal/PackageResolver.groovy

```diff
@@ -62,6 +66,43 @@ class PackageResolver {
     resolvePlayServiceRepository()
   }
 
+  def resolveTools() {
+    if (!folderExists(toolsDir)) {
+      throw new StopExecutionException('SDK tools not found!')
+    }
+
+    def sourcePropsFile = new File(toolsDir, 'source.properties')
+    if (!sourcePropsFile.canRead()) {
+      throw new StopExecutionException('Could not read source.properties inside SDK tools dir!')
+    }
+
+    def props = new Properties()
+    props.load(new FileInputStream(sourcePropsFile))
+
+    def revision = props.getProperty('Pkg.Revision')
+    if (revision == null) {
+      throw new StopExecutionException('Could not get property Pkg.Revision from SDK tools!')
+    }
+
+    def revisionParts = revision.split('\\.')
+    if (revisionParts == null || revisionParts.length == 0) {
+      throw new StopExecutionException('Could not handle property Pkg.Revision from SDK tools!')
+    }
+
+    // TODO how to replace hard-coded constant?
```

Comment:
FYI, my PR #111 avoids the need for a hard-coded constant by looking at the version from the list of available updates.